authors/apps/profiles/urls.py

```diff
@@ -1,6 +1,7 @@
 from django.urls import path
-from .views import RetrieveProfileAPIView
+from .views import RetrieveProfileAPIView, RetrieveAuthorsAPIView
 
 urlpatterns = [
-    path('profiles/<username>/', RetrieveProfileAPIView.as_view(), name='profile')
+    path('profiles/<username>/', RetrieveProfileAPIView.as_view(), name='profile'),
+    path('authors/', RetrieveAuthorsAPIView.as_view(), name='authors')
```

Comment:
let's maintain consistency here. Are we going to stick with users or use authors?

Reply:
Choose either to refer to our users as users or authors and let it be reflected in the endpoints. We already have urls names users. Are users separate from authors? If not, let's maintain one.

Reply:
Do you mean I should modify the way the name of the url path? They are not separate from authors